app/javascript/css/main.scss

```diff
@@ -515,3 +515,54 @@ a.disabled {
   color: var(--primary-color);
   border-bottom: 1px dashed var(--primary-color);
 }
+
+.delete-icon {
+  color: var(--status-red);
+}
+
+.inbox-sender {
+  font-size: 16px;
+  font-weight: 600;
+}
+
+.tabs {
```

Comment:
can we use bootstrap for these margins?

Reply:
so these were pulled from dbh repo with a little modification (https://github.com/ideacrew/enroll/pull/4031#discussion_r1682000321). On top of this, the mb-# options don't quite get to the specific measurements needed (mb-2 is 8px, which is the closest option). If there are other bootstrap classes that can get there I'd be open to it